exercises/two-fer/metadata.toml

```diff
@@ -1,3 +1,3 @@
-title = "Two-fer"
+title = "Two Fer"
```

Comment:
I feel a little unsure about this once, since it seems to imply that "fer" is a word, but it's difficult for me to really say that it is one.

Reply:
It's definitely not a word, but to me `Two Fer` isn't any less wrong than `Two-fer`, and I prefer the former as a title.

Reply:
I don't think my having made it up is a good reason, because I didn't put any thought into orthography when I made it.

Given the data about usage outside of Exercism I am leaning more heavily towards the hyphenated version of the title.

Reply:
Thank you @ee7. Those are good enough arguments that I don't feel like choosing `Two-Fer` over `Two Fer` is random.